example-widget-mui/src/ThemePage/ThemePage.tsx

```diff
@@ -774,6 +776,96 @@ export function WidgetListItem({
   );
 }
 
+export function AvatarsDemo() {
+  return (
+    <Box m={1}>
+      <Typography variant="h2">Avatars</Typography>
+      <p>Avatars can be used together with users or rooms.</p>
+      <p>
+        <ElementAvatar userId="@user:matrix.org" /> Avatar without image and
+        name
+      </p>
+      <p>
+        <ElementAvatar userId="@user:matrix.org" displayName="Display Name" />
+        Avatar without image
+      </p>
+      <p>
+        <ElementAvatar
+          userId="@user:matrix.org"
+          displayName="Display Name"
+          avatarUrl="https://example.com/test.png"
+        />
+        Avatar with broken image URL
+      </p>
+      <p>
+        <ElementAvatar
+          userId="@user:matrix.org"
+          displayName="Display Name"
+          avatarUrl="https://example.com/test.png"
+        />
+        Avatar with broken image URL
+      </p>
+      <p>
+        <ElementAvatar
+          userId="@oliver.sand:matrix.org"
+          displayName="Oliver Sand"
+          avatarUrl="mxc://matrix.org/KbhRFOIJekHQpMVIZGpuNxBG"
+        />
+        Avatar with mxc: URL (might not resolve)
+      </p>
+      <p>
+        <ElementAvatar
+          userId="@oliver.sand:matrix.org"
+          displayName="Oliver Sand"
+          avatarUrl="https://matrix-client.matrix.org/_matrix/media/r0/download/matrix.org/KbhRFOIJekHQpMVIZGpuNxBG"
+        />
+        Avatar
+      </p>
+      <p>
+        <ElementAvatar
+          userId="@oliver.sand:matrix.org"
+          displayName="Oliver Sand"
+          avatarUrl="https://matrix-client.matrix.org/_matrix/media/r0/download/matrix.org/KbhRFOIJekHQpMVIZGpuNxBG"
+          sx={(theme) => ({
+            outline: `${theme.palette.primary.main} solid 2px`,
+            border: `2px solid ${theme.palette.background.default}`,
+          })}
+        />
+        Avatar with outline
+      </p>
+      <p>
+        <ElementAvatar
+          userId="@oliver.sand:matrix.org"
+          displayName="Oliver Sand"
+          avatarUrl="https://matrix-client.matrix.org/_matrix/media/r0/download/matrix.org/KbhRFOIJekHQpMVIZGpuNxBG"
+          sx={{ opacity: 0.5 }}
+        />
+        Avatar with transparency
+      </p>
+      <p>
+        <ElementAvatar
+          userId="@oliver.sand:matrix.org"
+          displayName="Oliver Sand"
+          sx={{ width: 32, height: 32, fontSize: 25 }}
```

Comment:
The font size is not applied because the specificity of it is not correct. Is it possible to fix that in `styled`?

Reply:
Yeah, this is correct, I don't have a good solution for that. But I can remove it here from the example. Or one uses a more complex selector here to apply it.

Reply:
A complex selector works to set the font size, updated the PR.